### DISCUSSION TITLE
Band Tags missing

### DISCUSSION THREAD

**Discussion by ismailerradi** [corteva/rioxarray#525] [Q&A]
I started working on a project with rasters recently, especially multiple band rasters (e.g : 192). It seems that when I open it with rasterio, I have a tag for each band which says what data layer are you working on (e.g: band 1 tag : weather data) but when I open it with rioxarray, it appears that the attributes are always the same for all the bands . I don't know if there's an issue at open with rioxarray or is this a known problem for the library ?

Here's the example with rasterio : 

<img width="549" alt="Capture d’écran 2022-05-12 à 01 21 32" src="https://user-images.githubusercontent.com/49719400/167963090-15c5de50-6e80-42cd-aad0-dae97709d161.png">

Here's the example with rioxarray:

<img width="566" alt="Capture d’écran 2022-05-12 à 01 21 52" src="https://user-images.githubusercontent.com/49719400/167963153-f4f5cdd8-a7b1-4d37-bf9e-83c03385e456.png">

Thanks 👍

**Comment by snowman2** [comment]
See #296

**Reply by snowman2** [reply]
See #600
